### PR TITLE
impove randomness

### PR DESCRIPTION
**Pull request by albert-ziegler**
I've noticed `Math.random()` used to generate a secret. However, its values are not safe against prediction, so if I understand the context correctly, it's probably preferable to use `Window.crypto` instead.

Full disclosure: I noticed this using [LGTM.com](https://lgtm.com/projects/g/hakimel/reveal.js/alerts/?mode=list&result_filter=0e00f5fa7b93849767c1677dc71d9400f85c6a54), a code analysis platform which I also work for.

### CODE REVIEW

File: plugin/multiplex/index.js [hunk 1]
```diff
@@ -44,7 +44,8 @@ app.get("/", function(req, res) {
 
 app.get("/token", function(req,res) {
 	var ts = new Date().getTime();
-	var rand = Math.floor(Math.random()*9999999);
+	var rand = new Uint32Array(1);
+	window.crypto.getRandomValues(array)
```
Comment (lirantal) [inline plugin/multiplex/index.js]: probably a typo @albert-ziegler? I think you meant to put `rand` in that function call

Comment (albert-ziegler) [reply plugin/multiplex/index.js]: Indeed. Thanks for spotting, @lirantal!

**Comment by hakimel** [comment]
Thanks!

**Comment by hakimel** [comment]
Had to revert this, `window.crypto` doesn't exist in Node.js so this threw an exception that prevented the multiplex server from working.

**Pull request merged**